src/plugins/topology/dockModel.cpp

```diff
@@ -132,3 +133,25 @@ void DockModel::reload( const QModelIndex &index1, const QModelIndex &index2 )
 {
   emit dataChanged( index1, index2 );
 }
+
+DockFilterModel::DockFilterModel( ErrorList &errorList, QObject *parent = nullptr )
+  : mDockModel( new DockModel( errorList, parent ) )
```

Comment:
```suggestion
DockFilterModel::DockFilterModel( ErrorList &errorList, QObject *parent = nullptr )
  : QSortFilterProxyModel( parent )
  , mDockModel( new DockModel( errorList, this ) )
```

and remove the manual deleteLater call in ~DockFilterModel